src/components/ShareBlock/ShareModelBlock.js

```diff
@@ -4,13 +4,11 @@ import { STATES } from 'enums';
 
 import EmbedPreview from './EmbedPreview';
 
-const ShareModelBlock = ({ condensed, location, county }) => {
-  const locationName = STATES[location];
+const ShareModelBlock = ({ condensed, stateId, county }) => {
```

Comment:
You changed this, but not the code that references it in Embed.js.

So if you go to http://localhost:3000/us/wa and click the "EMBED" share button, and then click the share button inside the embed iframe, you get an error.

More stuff TypeScript could catch. :-)

Reply:
Share buttons inside share button  🤦  Thanks for the careful review!